### PR TITLE
Add additionalLabels for missing resources

### PR DESCRIPTION
**Pull request by usamaahmadkhan**

### CODE REVIEW

**Comment by stakater-user** [comment]
@usamaahmadkhan Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/d0b26f5e5ed4c6b004a5de0f5920f5b0e6523ca5/checks) has Failed!

**Comment by stakater-user** [comment]
@hanzala1234 Yikes! You better fix it before anyone else finds out! [Build](https://github.com/stakater-charts/application/commit/d0b26f5e5ed4c6b004a5de0f5920f5b0e6523ca5/checks) has Failed!

File: application/values-test.yaml [hunk 1]
```diff
@@ -244,7 +244,10 @@ ingress:
   hosts:
     - chart-example.local
     - '{{ .Release.Name }}.stakater.com'
-  
+
+  # Additional labels for this Ingress
+  additionalLabels:
+
```
Comment (hanzala1234) [inline application/values-test.yaml:250]: please add some labels here for testing, so this field does not remain empty and get validated

Comment (rasheedamir) [reply application/values-test.yaml:250]: @usamaahmadkhan ^^

**Comment by stakater-user** [comment]
@usamaahmadkhan Validation Successful

**Comment by hanzala1234** [comment]
dont merge now until commit history is cleaned

**Comment by stakater-user** [comment]
@Nebojsa92 Validation Successful

**Review by hanzala1234 — DISMISSED** [review]

**Review by hanzala1234 — APPROVED** [review]

**Comment by stakater-user** [comment]
@hanzala1234 Validation Successful
